### ISSUE TITLE
Add Props tech specification to ReliDocs

### ISSUE THREAD

**Issue by davidzbiral** [DISSINET/InkVisitor#1689]
There is a property chapter allready - add the JSON tech specification as a special section.

**Comment by Ptrhnk** [comment]
@davidzbiral It seem to me like the JSON tech specification is there. Is that still relevant or am I missing something?
[https://docs.religionistika.phil.muni.cz/books/from-texts-to-structured-data-building-knowledge-graphs-through-computer-assisted-semantic-text-modelling-castemo/page/properties-overview](https://docs.religionistika.phil.muni.cz/books/from-texts-to-structured-data-building-knowledge-graphs-through-computer-assisted-semantic-text-modelling-castemo/page/properties-overview)

**Comment by davidzbiral** [comment]
@Ptrhnk Yes, it seems that you guys did props, so let us no go towards Relations (another issue) and References! Closing this one.
